File: stdin_primer/python/06-03_integer_number_step3.py
# 1 行目で与えられる N 個の整数の入力 (paizaランク D 相当)
# https://paiza.jp/works/mondai/stdin_primer/stdin_primer__integer_number_step3

# 解答例
# 入力
n, *ary = map(int, input().split())
# 出力
[print(num) for num in ary]

File: stdin_primer/haskell/06-03_integer_number_step3.hs
-- 1 行目で与えられる N 個の整数の入力 (paizaランク D 相当)
-- https://paiza.jp/works/mondai/stdin_primer/stdin_primer__integer_number_step3

toInt :: String -> Int
toInt x = read x :: Int

toStr :: Int -> String
toStr x = show x :: String

main :: IO ()
main = do
  n : a <- map toInt . words <$> getLine

  putStr $ unlines $ map toStr a
